README.md

```diff
@@ -26,14 +26,21 @@ More on `create-react-app` [here](https://reactjs.org/docs/create-a-new-react-ap
 
 ### Auth
 
-To use FHIR servers that require auth, set the `REACT_APP_KF_USER` and `REACT_APP_KF_PW`
+To authenticate, the user can either use the UI login screen, or set environment
+variables. To use environment variables, set the `REACT_APP_KF_USER` and `REACT_APP_KF_PW`
 environment variables to the appropriate username and password, respectively. The
 app will use this and Basic Auth to make requests.
 
 ### FHIR APIs
 
 Creating a data dashboard over the FHIR API standard means that a user potentially
 has the ability to search over multiple datasets. This application can be launched
-over any FHIR server. To use a specific FHIR server, set the `REACT_APP_FHIR_API`
-from the command line to the base url of the server. An empty variable will default
-to the Kids First FHIR server.
+over any FHIR server. To use a specific FHIR server, set the following
+environment variables (also found at `.env.example`):
+
+`REACT_APP_FHIR_API`: The url at which the FHIR server is located<br>
+`REACT_APP_FHIR_API_NAME`: The display name of the FHIR server<br>
+`REACT_APP_FHIR_AUTH_REQUIRED`: `true` if the server requires auth (Basic), otherwise `false`
```

Comment:
This doesn't seem to work right now. Here are my env vars:
```
REACT_APP_FHIR_API_NAME=Localhost
REACT_APP_FHIR_AUTH_REQUIRED=true
REACT_APP_FHIR_API=http://localhost:8000/
REACT_APP_KF_USER=admin
REACT_APP_KF_PW=password
```
And when I launch the app it goes to: 
![image](https://user-images.githubusercontent.com/10283712/78044702-07dabc00-7343-11ea-9867-7c4c0ce09387.png)

Probably because the requires auth var didn't get picked up for some reason:
![image](https://user-images.githubusercontent.com/10283712/78044780-1de87c80-7343-11ea-966d-c7364d77f039.png)

After I edit the Localhost server config and switch to requires auth, it then asks me to login, and I'm able to auth successfully and see the home page.

Reply:
However, when I try to switch servers to `Kids First` it fails on a 401 Unauthorized because its still trying to use the Localhost server auth settings:

![image](https://user-images.githubusercontent.com/10283712/78045461-f940d480-7343-11ea-928f-7e5dc1aae7d7.png)

The request base64 encoded auth value: `YWRtaW46cGFzc3dvcmQ=` decodes to `admin:password`